emily/handler/src/api/handlers/chainstate.rs

```diff
@@ -41,10 +34,7 @@ pub async fn get_chain_tip(context: EmilyContext) -> impl warp::reply::Reply {
         // TODO(390): Handle multiple being in the tip list here.
         let api_state = accessors::get_api_state(&context).await?;
         let chaintip: Chainstate = api_state.chaintip().into();
-        Ok(with_status(
-            json(&(chaintip as GetChainstateResponse)),
-            StatusCode::OK,
-        ))
+        Ok(with_status(json(&(chaintip as Chainstate)), StatusCode::OK))
```

Comment:
The `as` cast seems unnecessary, given the line above. Same for a few of these functions.

Reply:
I'll fix that. Yeah it's unnecessary